test/integration/targets/aws_secret/aliases

```diff
@@ -1,3 +1,4 @@
 shippable/aws/group1
 cloud/aws
 posix/ci/cloud/group4/aws
+unsupported
```

Comment:
Mattclay pointed out this morning that there are some issues adequately resource restricting creating secrets. I'm going to look at addressing that after freeze for 2.8 and I don't want this module to miss the deadline so I've disabled the tests (which are passing locally for me with Python 2 and 3).